src/Extension/AbstractExtension.php

```diff
@@ -42,4 +42,11 @@ public function getOperators()
     {
         return [[], []];
     }
+
+    public function getLastModified(): int
+    {
+        $filename = (new \ReflectionClass($this))->getFileName();
+
+        return is_file($filename) ? filemtime($filename) : 0;
+    }
```

Comment:
Could we maybe check there if a Runtime exists ? 

Something very naive like 

```
if file_exists(str_replace('Extension.php', 'Runtime.php', $filename)
```

wdyt ?

Reply:
This would be a hard coded convention that matches with the Symfony implementation. https://github.com/symfony/symfony/tree/7.3/src/Symfony/Bridge/Twig/Extension

I would agree. Maybe using the class name instead?

Reply:
Yep!

A step further could be to "call" the extension methods (probaby just reflect them) to extract the various callable type info and see wich classes they are dependant from ?